Modules/Core/SpatialObjects/include/itkSpatialObjectToImageFilter.h

```diff
@@ -90,6 +90,18 @@ class ITK_TEMPLATE_EXPORT SpatialObjectToImageFilter : public ImageSource<TOutpu
   const InputSpatialObjectType *
   GetInput(unsigned int idx);
 
+  /** Generate an output image that matches the origin,
+   * size, direction, and spacing of this image. */
+  template <class TMatchImage>
+  void
+  SetMatchImage(TMatchImage * matchImage)
```

Comment:
The equivalent thing is called `ReferenceImage` in resample filter, it would be good if we kept that name here too.

Reply:
Fixed!